integrations/acquisition/covid_hosp/state_timeseries/test_scenarios.py

```diff
@@ -52,9 +53,9 @@ def test_acquire_dataset(self):
         self.test_utils.load_sample_dataset()
 
     # make sure the data does not yet exist
-    with self.subTest(name='no data yet'):
-      response = Epidata.covid_hosp('MA', Epidata.range(20200101, 20210101))
-      self.assertEqual(response['result'], -2)
+    # with self.subTest(name='no data yet'):
+    #  response = Epidata.covid_hosp('MA', Epidata.range(20200101, 20210101))
+    #  self.assertEqual(response['result'], -2)
```

Comment:
is this not passing any more?

Reply:
Actually this should be passing because of a fix I've added later (lines 40-41) - I'll uncomment this and see how it goes